lldb/source/Plugins/Process/gdb-remote/ProcessGDBRemote.h

```diff
@@ -301,7 +301,8 @@ class ProcessGDBRemote : public Process,
   using FlashRange = FlashRangeVector::Entry;
   FlashRangeVector m_erased_flash_ranges;
 
-  bool m_vfork_in_progress;
+  // Number of vfork in process.
```

Comment:
```
// Number of fork() or vfork() operations being handled.
```

Reply:
This is only used by vfork() not fork().